lib/rules/color-named/index.mjs

```diff
@@ -18,7 +18,7 @@ import validateOptions from '../../utils/validateOptions.mjs';
 const ruleName = 'color-named';
 
 const messages = ruleMessages(ruleName, {
-	expected: (expected, actual) => `Expected "${actual}" to be "${expected}"`,
+	expected: (actual, expected) => `Expected "${actual}" to be "${expected}"`,
```

Comment:
note

https://github.com/stylelint/stylelint/blob/0a5f75bc6ff3dea7978fddbb8a5da762691a2074/types/stylelint/index.d.ts#L418
i.e. for consistency with 8 other rules